docs/source/images/dask-overview_scheduler_border.svg

```diff
@@ -0,0 +1,58 @@
+<svg width="1920" height="1080" viewBox="0 0 1920 1080" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@jsignell this is for your border suggestion on [How to... Deploy Dask clusters](https://docs.dask.org/en/latest/how-to/deploy-dask-clusters.html). Maybe the box should go around the 'schedulers' text too?

Reply:
Yeah I think around the schedulers too and maybe it needs a bit of text on the page that just calls out that we are talking about the schedulers section and that the distributed overview below is kind of like a zoomed in graphic about just the distributed scheduler. I am thinking about how they do in maps:

![image](https://user-images.githubusercontent.com/4806877/146049847-1d88ac26-c984-4ce1-995d-037af5de1473.png)

Not saying that we need to make something like that. I just think the graphics would be more easy to understand if we mention how they relate.

Reply:
Yup, that makes sense! I can adjust this a bit.